README.md

```diff
@@ -21,51 +21,69 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
-	- [Networking](#networking)
-	- [PaaS](#paas)
-	- [Remote Container Manager / Orchestration](#remote-container-manager--orchestration)
-	- [Reverse Proxy](#reverse-proxy)
-	- [Security](#security)
-	- [Serverless](#serverless)
-	- [Service Discovery](#service-discovery)
-	- [CaaS - Services for running containers](#services-for-running-containers)
-	- [Terminal User Interface](#terminal-user-interface)
-	- [Testing](#testing)
-	- [Utilities](#utilities)
-	- [Volume management and plugins](#volume-management-and-plugins)
-	- [Web Interface](#web-interface)
+- Introduction
+  - [What is Docker?](#what-is-docker)
+  - [Where to start?](#where-to-start)
+  - [Where to start? (Windows)](#where-to-start-windows)
+- [Projects](#tools)
+	- Container Operations
+		- Container Composition
+		- [Deployment and Infrastructure](#deployment-and-infrastructure)
+		- [Monitoring](#monitoring--logging)
+		- [Networking](#networking)
+		- [Orchestration](#remote-container-manager--orchestration)
+		- [PaaS](#paas)
+		- [Reverse Proxy](#reverse-proxy)
+		- [Security](#security)
+		- [Service Discovery](#service-discovery)
+		- [Volume management / Data](#volume-management-and-plugins)
+		- User Interface
+			- [Terminal](#terminal-user-interface)
+			- Native
+			- [Web](#web-interface)
+	- Docker Images
+		- [Builder](#image-builder)
+		- [Dockerfile](#dockerfile)
+		- [Linter](#linter--validator)
+		- Inside Container Tools
+		- [Registry](#hosting-images-registries)
+	- Development with Docker
+		- [API Client](#developer-tools)
+		- [CI/CD](#continuous-integration--continuous-delivery)
+		- [Development Environment](#development-environments)
+		- [Garbage Collection](#garbage-collection)
+		- [Serverless](#serverless)
+		- [Testing](#testing)
+		- [Wrappers](#local-container-manager)
+		- [Utilities](#utilities) <- must be split
+	- Legacy tools
+	- Services based on Docker ($$$)
+		- [CI/CD](#ci-services)
+		- [CaaS - Services for running containers](#services-for-running-containers)
+		- [Monitoring](#monitoring--logging-services)
+		- Registries
+		- PaaS
 - [Useful Resources](#useful-resources)
+	- Awesome Lists
+	- [Communities and Meetups](#communities-and-meetups)
+		- [Brazilian](#brazilian)
+		- [Chinese](#chinese)
+		- [English](#english)
+		- [Russian](#russian)
 	- [Good Tips](#good-tips)
+	- [Raspberry Pi / ARM](#raspberry-pi--arm)
 	- [Newsletter](#newsletter)
 	- [Security](#security-1)
-	- [Raspberry Pi / ARM](#raspberry-pi--arm)
-- [Videos](#videos)
-	- [Main Account](#main-account)
-	- [Useful videos](#useful-videos)
-- [Interesting Twitter Accounts](#interesting-twitter-accounts)
-- [Communities and Meetups](#communities-and-meetups)
-	- [Brazilian](#brazilian)
-	- [Chinese](#chinese)
-	- [English](#english)
-	- [Russian](#russian)
-
+	- [Videos](#videos)
+		- [Main Account](#main-account)
+		- [Useful videos](#useful-videos)
+	- ~~[Interesting Twitter Accounts](#interesting-twitter-accounts)~~
```

Comment:
I also agree on removing Twitter links, but at least some blogs should remain e.g. The New Stack. In addition, blog sites have links to their social media accounts.